templates/careers/careers_base.html

```diff
@@ -88,7 +88,7 @@
       </select>
     </div>
     <div class="col-9 col-start-large-5">
-      {% if (request.path != "/careers/lifestyle" and request.path != "/careers/ethics" and request.path != "/careers/travel" and request.path != "/careers/progression" and request.path != "/careers/diversity") %}
+      {% if (request.path != "/careers/lifestyle" and request.path != "/careers/ethics" and request.path != "/careers/travel" and request.path != "/careers/progression" and request.path != "/careers/diversity" and request.path != "/careers/job-detail") %}
```

Comment:
I think this block is starting to get a little unwieldy, it's not too bad at the moment but if more sections are added it'll get harder and harder to parse. Maybe we could create a template filter that compares the request path to an array of values, and if there's a match, we don't render the tabs.

Reply:
@sowasred2012 fair comment. Very likely I will need to re-factor this part. Thanks